src/Card/Card.tsx

```diff
@@ -14,20 +16,28 @@ const DEFAULT_CARD_HEIGHT = 130;
 
 const StyledCard = styled(MuiCard, {
   shouldForwardProp: (prop) => prop !== 'elevation' && prop !== 'fullWidth',
```

Comment:
I think you need to add some props here to remove warnings that they are not supported on the DOM. `isSelected` and `isOver` for example